### PR TITLE
[Motor-1175] Document Network Compression

### PR DESCRIPTION
**Pull request by caseyclements**
Adds a small section to the readthedocs Configuration page that links to the PyMongo pages for Network Compression, and also mentions optional dependencies.

### CODE REVIEW

**Review by ShaneHarvey — APPROVED** [review]
LGTM!

File: doc/configuration.rst [hunk 1]
```diff
@@ -46,3 +46,17 @@ system; to override the default set the environment variable ``MOTOR_MAX_WORKERS
...
+Like PyMongo, Motor supports network compression where network traffic between
+the client and MongoDB server are compressed.
+Keyword arguments to the Motor clients match those in MongoClient, documented
+`here <https://pymongo.readthedocs.io/en/stable/examples/network_compression.html>`_.
+By default no compression is used. If you wish to use wire compression,
+you will have to install one of the optional dependencies.
+Snappy requires `python-snappy <https://pypi.org/project/python-snappy>`_
+and zstandard requires `zstandard <https://pypi.org/project/zstandard>`_::
+
+  $ python3 -m pip install "pymongo[snappy, zstd]"
```
Comment (ShaneHarvey) [inline doc/configuration.rst:62]: pymongo -> motor

**Review by ShaneHarvey — COMMENTED** [review]
noticed one minor change.

File: doc/configuration.rst (hunk 1 above)
Comment (caseyclements) [reply doc/configuration.rst:62]: Missed one. Thanks!

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**